### PR TITLE
Youtube Project - First Pull Request

### PR DESCRIPTION
**Pull request by tiagoes**
Please review our first Project

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+}
+#languageH{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  background: #167ac6;
+  color: white;
+  width: 100%;
+  /**/
```
Comment (HADDADSOHAIB) [inline index.css]: Delete unnecessary comments to keep your code clean.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+  padding: 15px;
+}
+#vidTitle h1{
+  font-size: 20px;
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+#section2 {
+  /* border: 1px solid black; */
```
Comment (HADDADSOHAIB) [inline index.css]: Delete commented code:
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: index.css [hunk 3]
```diff
@@ -0,0 +1,569 @@
...
+#odin  {
+  /* border: 1px solid black; */
+  margin-left: -5px;
+  margin-bottom: 25px;
+  color: rgb(114, 112, 112);
+  font-weight: 600;
+}
+#thirdFlex  {
+  /* border: 1px solid black; */
+  display: flex;
```
Comment (HADDADSOHAIB) [inline index.css]: also here delete commented code:
- [x] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @tiagoes and @defoebrand   :handshake: 

### Good job so far :clap:  your project is looking good :medal_sports: 

### there are some changes to get the project approved :point_down:
- I think your readme could be improved, the readme file should be descriptive, add more information about the project and fix the live demo link, it is not working (suggesting: use http://raw.githack.com/ to generate a live link)
- Your pull request description could also be improved, always remember to be descriptive, write what you did in your feature and what this pull request is adding, take a look at this link to understand why a GitHub repository should look professional:
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
- You should check the project with the validator and fix the issues:
- [x] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. (Students were asked to use this one, so in most cases their code should be already validated) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)
Screenshot from 2020-06-05 02-40-25
- Your css file needs to be improved, take a second look at this article and try to avoid bad practice in your code:
- [x] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
check the comments for additional explanation.
- The search bar should be centered:
- [x] The search bar is aligned to the middle of the navbar ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/searchbar_align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
- the share and subscribe should be buttons:
- [x] It has a share button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
- [x] It has a subscribe button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_subscribe_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
### Please do the changes and ask for another code review :+1:

**Comment by tiagoes** [comment]
**We made the requested changes:** 

Increased the description in the readme file.
Fix the Live link.
We checked the HTML market with the validator 
Deleted excess comments in CSS.
Center aligned the search bar.
Made Share and Subscribe a button.

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @tiagoes & @defoebrand  :wave:
-
You have done a splendid job! :tada: Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
Please makes changes and submit another review request.

- [ ] Delete all the necessary branches in this repo they are cluttering your awesome project's repo.
- [ ] The markup is still showing some error in the validator. Kindly fix that, thank you.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by tiagoes** [comment]
We made the requested changes:
We correct the error showed in the Markup
We deleted the extra git branches

**Comment by peterrobert** [comment]
### PULL REQUEST UNDER REVIEW BY PETER ROBERT

**Review by peterrobert — APPROVED** [review]
### APPROVED :+1: 
Great job. your project looks exactly like the clone :+1: a wonderful job . Great job implementing the suggested changes.
**on to the next project**
### HAPPY CODING

**Pull request merged**